### ISSUE TITLE
[Feature request] Add option to browser context menu when right-clicking link "Skip Redirector"

### ISSUE THREAD

**Issue by cocoonkid** [einaregilsson/Redirector#288]
[Feature request] Add option to browser context menu when right-clicking link "Skip Redirector"

That would be amazing. Thanks

**Comment by einaregilsson** [comment]
Sounds like a good idea, I might well add it one day 🙂

**Comment by Gitoffthelawn** [comment]
That *is* an interesting idea.  I'm trying to recall specifics cases when that would have come in handy.  For years, I've always temporarily disabled Redirector and then re-enabled it, which obviously requires more steps.

**Comment by einaregilsson** [comment]
I guess it would be fairly easy by adding a content script to add the menu item, then when you click it send the url to the background process and it just knows to ignore the next request for exactly that url.

**Comment by Gitoffthelawn** [comment]
Yes, I think it would be fairly straightforward.  The only catch I see is deciding how it would handle recursive Redirector rules (the output of a rule being processed through the rules again).  I think recursion is currently disabled, but it's the one piece of functionality that I hope you have time to implement before Iceland becomes the next Sahara. :)

**Comment by cocoonkid** [comment]
You beautiful people are awesome. Thanks for thinking about how to make this happen!
❤️
